### PR TITLE
happy new year

### PR DESCRIPTION
**Pull request by janl**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -187,7 +187,7 @@
      same "printed page" as the copyright notice for easier
      identification within third-party archives.
 
-  Copyright [yyyy] [name of copyright owner]
+  Copyright 2019 The Apache Foundation
```
Comment (kxepal) [inline LICENSE]: The license itself provides a template here. Should we edit it?

**Comment by kxepal** [comment]
:santa: :christmas_tree: :gift: :fireworks: :tada:

File: LICENSE (hunk 1 above)
Comment (janl) [reply LICENSE]: Yup ;)

**Comment by jiangphcn** [comment]
+1

**Pull request merged**